x-pack/test/spaces_api_integration/common/fixtures/es_archiver/saved_objects/spaces/data.json

```diff
@@ -273,7 +272,9 @@
       ],
       "type": "visualization",
       "updated_at": "2017-09-21T18:49:16.270Z",
-      "namespaces": ["space_1"]
+      "namespaces": ["space_1"],
+      "coreMigrationVersion": "8.8.0",
+      "typeMigrationVersion": "8.5.0"
```

Comment:
I have added the `coreMigrationVersion` and `typeMigrationVersion` for all **actual** saved objects.
Not doing so causes all the "core migrations" to run (as if it was a really old SO), and leads to the `"namespaces": ["space_1"]"` to be incorrectly handled by `internal_transforms.ts`, resulting in `"namespaces": ["default"]`.